app.py

```diff
@@ -4,9 +4,42 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+def toggle_string(status_string):
+    if status_string == 'Not Started':
+        return 'Completed'
+    return 'Not Started'
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    return render_template('index.html', items=items)
+
+@app.route('/sorted/<field>')
+def sorted_by(field):
+    if field == 'status':
+        reverse = True
+    else:
+        reverse = False
+    sorted_items = sorted(session.get_items(), key=lambda item: item[field], reverse=reverse)
+    return render_template('index.html', items=sorted_items)
+
+@app.route('/add', methods=['POST'])
+def add_something():
+    session.add_item(request.form.get('title'))
+    return index()
```

Comment:
This ends up leaving the url as http://127.0.0.1:5000/add
(This means somebody refreshing the page could accidentally repeat the add)

Can you think of a solution that would preserve the original url?